src/modules/languages/dotnet.nix

```diff
@@ -6,11 +6,21 @@ in
 {
   options.languages.dotnet = {
     enable = lib.mkEnableOption "tools for .NET development";
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.dotnetCorePackages.sdk_7_0;
```

Comment:
According to the [manual](https://nixos.org/manual/nixpkgs/unstable/#dotnetcorepackages.sdk-vs-dotnetcorepackages.runtime-vs-dotnetcorepackages.aspnetcore):
> `dotnetCorePackages.sdk` contains both a runtime and the full sdk of a given version

Should be a good default package, however, we don't have a generic `dotnetCorePackages.sdk` package pointing to the latest release. This is why I set 7.0 as the default one.